cypress/features/regression/carousel.feature

```diff
@@ -1,14 +1,9 @@
 Feature: Carousel component
   I want to check Carousel properties
 
-  @positive
-  Scenario: Initial slide title is Slide 3
```

Comment:
why did you remove it ? I don't see it's covered by chromatic

Reply:
here is an example :) https://www.chromatic.com/test?appId=5ecf782fe724630022d27d7d&id=5fdb5c4c0d86c5002104c5b9